File: flipper.py
import tkinter as tk
import random

root = tk.Tk()
root.title("My color flipper")
root.geometry("400x300")
# Create a label for your window
label = tk.Label(root, test = "Click the button to flip color", font=("Arial", 16))
label.pack(pady = 30) #pack places the label in the window and paddy adds vertical space

root.mainloop
# colors[] = ["green", "pink", "red", "white", "blue", "green"]

File: flipper.js
const colors = [
    "red",
    "blue",
    "green",
    "yellow",
    "purple",
    "orange",
    "pink",
    "cyan",
    "lime",
    "violet"
]
//event listeners
const label = document.getElementById("colorlabel");
const button = document.getElementById("flipbutton");

// function to pick random colors
function flipcolor(){
    // pick a random position from the array
    const randomindex = Math.floor(Math.random() * colors.length);
    // get the value at that position
    const color = colors[randomindex];
    // change background
    document.body.style.backgroundColor = color;
    // change text label
    label.textContent = "Current color: " + color;

}
button.addEventListener("click", flipcolor);